packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.spec.ts

```diff
@@ -46,7 +78,8 @@ describe("PrettyPrintComponent", () => {
 		'  <menuitem value="New" onclick="CreateNewDoc()" />',
 		'  <menuitem value="Open" onclick="OpenDoc()" />',
 		'  <menuitem value="Close" onclick="CloseDoc()" />',
-		"</menu>"
+		"</menu>",
+		""
```

Comment:
Is this empty string really needed in the raw data now?

Reply:
isn't it because prettier want the file ends with newline ?

Reply:
@nicanac is right. Prettier end with a newline. Without this, our tests don't pass.